neuprint/client.py

```diff
@@ -489,7 +531,24 @@ def _fetch_raw(self, url, json=None, ispost=False):
     def _fetch_json(self, url, json=None, ispost=False):
         r = self._fetch(url, json=json, ispost=ispost)
         return ujson.loads(r.content)
+    
+    def _fetch_arrow(self, url, json=None, ispost=False):
+        r = self._fetch(url, json=json, ispost=ispost)
+        content_type = r.headers.get('Content-Type', '')
+        if 'application/vnd.apache.arrow.stream' not in content_type:
+            raise Exception(f"Expected Arrow stream content type but got: {content_type}")
+        reader = pa.ipc.open_stream(pa.py_buffer(r.content))
+        table = reader.read_all()
+
+        # Convert to pandas DataFrame
+        df = table.to_pandas()
 
+        # Process each column - convert Map types to Python dictionaries
+        for col in df.columns:
+            # Apply the conversion function to each value in the column
+            df[col] = df[col].map(lambda x: self._convert_mapvalue_to_dict(x) if x is not None else None)
+
+        return df
```

Comment:
```suggestion
        return reader.read_all().to_pandas(maps_as_pydicts='strict')

```

Reply:
Made change and tested it against new arrow-enabled neuprinthttp successfully. However requires modifying the environment to include PyArrow>=13 although I was able to pip upgrade to PyArrow 20 even with Python 3.9.7. @stuarteberg

Reply:
I'll add it then to the README.md for this round of changes. Conda is the first install method suggested and might be broken as is for PyArrow.

Reply:
This command also seems to work, at least on linux:

```bash
conda create -n test-neuprint-python python=3.9 'neuprint-python>=0.5.1' 'pyarrow>=20' 'numpy>=2' 'pandas>=2'
```